### PR TITLE
f-checkout@v0.97.0 - Changed main headers to be h1 elements

### PR DESCRIPTION
**Pull request by 1itvinka**
### Changed
- Changed main headers to be h1 elements

I've made sure that the text still looks the same

### CODE REVIEW

File: packages/components/organisms/f-checkout/src/components/Header.vue [hunk 1]
```diff
@@ -108,6 +110,10 @@ export default {
     text-align: center;
 }
 
+.c-checkoutHeader-userTitle {
```
Comment (1itvinka) [inline packages/components/organisms/f-checkout/src/components/Header.vue]: not sure if this class name is good enough

Comment (charliearlie) [reply packages/components/organisms/f-checkout/src/components/Header.vue]: I think if this is the H1 asking the user to confirm their details then it's definitely good enough 🙂

**Review by charliearlie — DISMISSED** [review]

**Review by dandel10n — DISMISSED** [review]

**Review by dandel10n — DISMISSED** [review]

**Review by charliearlie — DISMISSED** [review]

File: packages/components/organisms/f-checkout/src/components/Header.vue (hunk 1 above)
Comment (ashleynolan) [reply packages/components/organisms/f-checkout/src/components/Header.vue]: Any reason not to just call it `c-checkoutHeader-title`? Just protects it a little if the content of the h1 changes in the future (away from being a user title)

Comment (ashleynolan) [reply packages/components/organisms/f-checkout/src/components/Header.vue]: Just seen that there's already a title called `c-checkoutHeader-title` so guessing that's why. Looking at the two titles, I'd probably make this class the `c-checkoutHeader-title` and change the classname of the other to be named closer to guest experience – `c-checkoutHeader-guestTitle`?

Comment (1itvinka) [reply packages/components/organisms/f-checkout/src/components/Header.vue]: yeah, that was the reason. The styles are a bit different so I needed a new class name. Your suggestion sounds good. Will change the class names.

**Review by DamianMullins — DISMISSED** [review]

**Review by dandel10n — DISMISSED** [review]

File: packages/components/organisms/f-checkout/src/components/Header.vue [hunk 2]
```diff
@@ -23,9 +25,9 @@
         <div
             v-else
             data-test-id="guest-checkoutHeader">
-            <h2 :class="$style['c-checkoutHeader-title']">
+            <h1 :class="$style['c-checkoutHeader-guestTitle']">
```
Comment (siggerzz) [inline packages/components/organisms/f-checkout/src/components/Header.vue]: Could we add a `data-test-id="user-title"` attribute here too?

Comment (1itvinka) [reply packages/components/organisms/f-checkout/src/components/Header.vue]: I'll add it now

**Review by dandel10n — APPROVED** [review]

**Review by DamianMullins — APPROVED** [review]

**Pull request merged**